backend/src/main/resources/freinet_import/import_xml.txt

```diff
@@ -0,0 +1 @@
+https://www.lbe.bayern.de/engagement-anerkennen/ehrenamtskarte/akzeptanzstellen/app-daten.xml
```

Comment:
I think this could be made confiigurable via the System properties through gradle like the database connection.

Reply:
Also this file should not be in the freinet_import directory

Reply:
You are right 👍